src/v2/Apps/Settings/Routes/MyCollection/Components/SettingsMyCollectionArtworks.tsx

```diff
@@ -62,8 +76,31 @@ const SettingsMyCollectionArtworks: FC<SettingsMyCollectionArtworksProps> = ({
     handleClick(endCursor, page)
   }
 
+  const dismissMyCollectionMessage = () => {
+    window.localStorage.setItem("HAS_SEEN_MY_COLLECTION_MESSAGE_BANNER", "true")
+    setHasDismissedMessage(true)
+  }
+
   return (
     <>
+      {!hasDismissedMessage && (
+        <Message
+          variant="info"
+          mt={-SETTINGS_ROUTE_TABS_MARGIN}
+          mb={SETTINGS_ROUTE_TABS_MARGIN}
+        >
+          <Flex flexDirection="row" justifyContent="space-between">
+            <Box />
+            <Text mx={1}>
+              Access all the My Collection features on the Artsy app. Coming
+              soon also on web.
+            </Text>
+            <Clickable onClick={dismissMyCollectionMessage}>
+              <CloseIcon />
+            </Clickable>
+          </Flex>
```

Comment:
You should avoid messing with the internals of this component so much and just add support for an `onDismiss` callback within the Palette component.

Reply:
@dzucconi We don't have a dismissible Message Component. And none of the Banner variants match the design.